src/main/java/com/yahoo/identity/Identity.java

```diff
@@ -20,8 +21,8 @@ public Identity(@Nonnull Storage storage) {
 
     @Nonnull
     public AccountService getAccountService() {
-        //TODO
-        return null;
+        AccountService sqlAccountService = new SqlAccountService(storage);
```

Comment:
- A same account service should be returned every time
- Instantiating the `SqlAccountService` will create a coupling between `Identity` and `SqlAccountService`. The `AccountService` should be injected into `Identity` (e.g. via constructor argument like `Storage`. The original `storage` constructor argument can be removed, shouldn't be used here. The `SqlAccountService` should be instantiated inside `DefaultIdentityFactory`. See https://github.com/sleepingpig/java_web_service_example/blob/master/src/main/java/com/vox/DefaultVoxFactory.java for example.